specs/eip4844/polynomial-commitments.md

```diff
@@ -110,37 +134,96 @@ def blob_to_kzg(blob: Blob) -> KZGCommitment:
 def verify_kzg_proof(polynomial_kzg: KZGCommitment,
                      x: BLSFieldElement,
                      y: BLSFieldElement,
-                     quotient_kzg: KZGProof) -> bool:
+                     kzg_proof: KZGProof) -> bool:
     """
     Verify KZG proof that ``p(x) == y`` where ``p(x)`` is the polynomial represented by ``polynomial_kzg``.
     """
     # Verify: P - y = Q * (X - x)
-    X_minus_x = bls.add(KZG_SETUP_G2[1], bls.multiply(bls.G2, BLS_MODULUS - x))
-    P_minus_y = bls.add(polynomial_kzg, bls.multiply(bls.G1, BLS_MODULUS - y))
+    X_minus_x = bls.add(bls.bytes96_to_G2(KZG_SETUP_G2[1]), bls.multiply(bls.G2, BLS_MODULUS - x))
+    P_minus_y = bls.add(bls.bytes48_to_G1(polynomial_kzg), bls.multiply(bls.G1, BLS_MODULUS - y))
     return bls.pairing_check([
         [P_minus_y, bls.neg(bls.G2)],
-        [quotient_kzg, X_minus_x]
+        [bls.bytes48_to_G1(kzg_proof), X_minus_x]
     ])
 ```
 
+#### `compute_kzg_proof`
+
+```python
+def compute_kzg_proof(polynomial: Sequence[BLSFieldElement], x: BLSFieldElement) -> KZGProof:
```

Comment:
I think it might be a good idea to document here that the polynomial is in evaluation form